src/renderer/Components/ReceivingPackage/NumberInput/index.jsx

```diff
@@ -1,24 +1,15 @@
+import { VALID_KEY } from "../../../constants/validKey";
```

Comment:
```suggestion
import { VALID_KEY } from "@renderer/constants/validKey";
```
path alias를 사용하실 수 있습니다! 😊👍

Reply:
수정 완료했습니다. 😁

![image](https://github.com/user-attachments/assets/62770bd2-8b96-46a3-9758-f86e4c3d466d)